### ISSUE TITLE
Building and publishing multi-arch images had an impact on project's CI

### ISSUE THREAD

**Issue by decko** [pulp/pulp-oci-images#565]
After getting some PRs where we build multi-arch images (amd64,arm64) our CI can't publish new images anymore, neither run the tests, like [here](https://github.com/pulp/pulpcore/actions/runs/6952326126/job/18915733298#step:9:449)

**Comment by StopMotionCuber** [comment]
I guess that's on me. I've just seen that the last fix I've submitted should have not only been applied to "Push app images to registries" but also to "Push base images to registries". Unfortunately it's not that easy debugging some of that stuff without the credentials to actually login to registries and push stuff in a different environment.

Contributions still welcome? 😬

**Comment by mdellweg** [comment]
At least we have some trail of information what went wrong. Sorry that we couldn't wait for it to be fixed in place. This really threw a wrench in the gearbox of our project wide CI's.
I believe we will be more careful next time and ideally we could catch these kind of issues in the PR's (not sure it's possible). I can only speak for myself: I'd still welcome the contribution.

**Comment by ipanova** [comment]
> Contributions still welcome? 😬

In development things go sideways all the time. Better having iterations with mistakes that'd eventually lead to success rather than a flat line  ![waveform-path-ecg](https://github.com/pulp/pulp-oci-images/assets/4253816/9d208d9c-773c-4ec5-9469-8cc5aca3b485)

**Comment by StopMotionCuber** [comment]
> > Contributions still welcome? 😬
> 
> In development things go sideways all the time. Better having iterations with mistakes that'd eventually lead to success rather than a flat line ![waveform-path-ecg](https://user-images.githubusercontent.com/4253816/285160155-9d208d9c-773c-4ec5-9469-8cc5aca3b485.svg)

That's true. I'm more comfortable if it goes sideways on my personal projects or work projects that I have control over than on upstream projects.

Anyway, fixing things there and will submit yet another PR when actions are fully running through. Deleted the login of Quay and Dockerhub and pushing to my personal container registry, so I have tested it end to end.
As a sidenode, when thinking I fixed everything, yet another bug appeared with `podman manifest push` needing 2 arguments. Not the case on my personal workstation, as that seemed to be changed with `podman 4.6.0`, which is not available in the Ubuntu 22.04 image.
